### ISSUE TITLE
Images and Icons

### ISSUE THREAD

**Issue by hone-zaun** [sajidali007522/edu#4]
* There are some images, which are not final at the moment.
* Like the Images for the badges. The icons in this image may also change in the future.
* We should not style the badge-earning widget as one background image. It would be better if we separate the badge itself from the background.
* Then we can use bulmas level or columns to easily prevent the Text from overlapping the badge itself. We also want to have such important images as inline image, because it makes them more accessible!
* If you need something for this task, please let me know.

* ![grafik](https://user-images.githubusercontent.com/51439140/76018764-eee81200-5f20-11ea-85ed-58516a22f6fd.png)
* 
![grafik](https://user-images.githubusercontent.com/51439140/76018776-f4455c80-5f20-11ea-9ac1-5bf3d9b60c4f.png)

**Comment by sajidali007522** [comment]
@hone-zaun  Thanks for asking in detail.
I think you using old version of html
Can you please download latest and then check it? 
For background image, I used full image like icon+background because designer provide me this kind of images and have list of images so you can simply change it if you ant 16,32,64 etc 

I did samething like you want last time but now it's full image and using inside the html instead of css background.

**Comment by hone-zaun** [comment]
Yeah, you are right. They are now inline. Thank you! I kinda missed that.

But I'm still not sure if it is a good idea to deliver them as a whole image instead of two separated ones.

So that we always use the same big greenish "background-image" and an other single image for the badge itself...

What do you think? Can you imagine something like that?
* Having this kind of image as the background
  * ![badge-section](https://user-images.githubusercontent.com/51439140/76103181-8ad15680-5fd1-11ea-8cf6-f506e2ad9a6b.png)
* And the single icon on top of it?
  * ![badge-10](https://user-images.githubusercontent.com/51439140/76103280-b5bbaa80-5fd1-11ea-9bc7-879ce8f6882e.png)

**Comment by sajidali007522** [comment]
@hone-zaun  
Yes you are right, but in my assets directory have list of big images like you sent screenshot
It contain all images so you can use any of it. 
And am 100% agree with you point that it should be two images instead of one large, I just did because I have only combine images. 
But if you still want to change then I will do it for you, I will ask designer to provide us images separate images then will create according to it.

**Comment by hone-zaun** [comment]
@sajidali007522 That would be great. We definitely need to separate these images.

But we should integrate the collar/sash to the badge as well. Otherwise the positioning of them will be messy.

* So we should make it like this:
  * The Background is pure `background-color: #3ebfb4;`. Please use a css-class for that.
  * The Badge is a single image with the collar/sash.
      * ![badge_single](https://user-images.githubusercontent.com/51439140/76217893-90b67a00-6213-11ea-981e-8730c455644d.png)
  *  And the small "stars" are images as well
      * ![grafik](https://user-images.githubusercontent.com/51439140/76218446-8d6fbe00-6214-11ea-80fd-c3853bda8653.png)
* This way we can control the position of the badge itself, change the color of the background if needed and don't blow up this section with unnecessary big images and loading times.
